android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -48,49 +52,11 @@ class PeerConnectionObserver implements PeerConnection.Observer {
         this.webRTCModule = webRTCModule;
         this.id = id;
         this.dataChannels = new HashMap<>();
-        this.localStreams = new ArrayList<>();
```

Comment:
I just remembered. Without this, how does MediaStream.release() work?

We need this custom API both for local streams and tracks to dispose the native side since we cannot latch on the JS GC to do it then.

Reply:
`WebRTCModule` stores a hashmap of local streams? here: 
```java
    final Map<String, MediaStream> localStreams;
```

Reply:
`mediaStream` functions inside `WebRTCModule` interact with that hashmap for the lifetime of mediaStreams.

Reply:
Awesome I had forgotten we store them there! 😅